lib/tmpl.js

```diff
@@ -186,4 +186,7 @@ var tmpl = (function() {
                   r.global ? 'g' : '')
   }
 
-})()
\ No newline at end of file
+})()
+
+if (typeof exports === 'object')
+  module.exports = tmpl
```

Comment:
@aurri want your feedback here. Maybe you have a different idea on how to package things.